cmd/gb-vendor/vendor/imports_test.go

```diff
@@ -0,0 +1,30 @@
+package vendor
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+func TestParseImports(t *testing.T) {
+	root := filepath.Join(getwd(t), "_testdata")
+
+	got, err := ParseImports(root)
+	if err != nil {
+		t.Fatalf("ParseImports(%q): %v", root, err)
+	}
+
+	want := set("fmt", "github.com/quux/flobble")
```

Comment:
The bug is the s.Path.Value returns quoted versions, we don't strip the quotes unless it does not match the standard library, which it never will, because quotes.